_episodes/08-func.md

```diff
@@ -918,6 +918,10 @@ readable code!
 > > `a` and `b` in this case) and trades their values. The function does not
 > > return any values and does not alter `a` or `b` outside of its local copy.
 > > Therefore the original values of `a` and `b` remain unchanged.
+> >
+> > Note that the recommended way of swapping the content of `a` and `b` in Python is
+> > `a, b = b, a`. You can see why this works in the
+> > [Additional Exercises]({{ page.root }}/extra_exercises/index.html).
```

Comment:
Based on the resolution of the issue #460, I am not sure we want to include this idiom. Also, with the challenge ultimately being about variable scope rather than how to swap variable values, it may be distracting.

Reply:
I was thinking about this overnight. From a distractions perspective the whole swapping functionality of the function may be distracting from the challenge itself. Maybe the **The Old Switcheroo** exercise should be replaced by a function that does not switch variables at all.

Reply:
I agree with you about the swapping being distracting; this seems something of a trick question. Looking at the other exercises based on your comment, I notice the challenge titled "Variables Inside and Outside Functions" seems to highlight the same concept of variable scope. Perhaps the Old Switcheroo could go away altogether in that case. Additionally, scanning over the episode material, I am not seeing us actually introduce the concept of variable scope or local variable...if that is the case, perhaps it is something we need to add... @maxim-belkin ?

Reply:
I agree that the **Variables Inside and Outside Functions** challenge teaches essentially the same content, and in a less cluttered way. As such, maybe **The Old Switcheroo** should indeed be removed. 

Maybe some of the explanatory text from the  **The Old Switcheroo** solution about local variables could be shifted over to the  **Variables Inside and Outside Functions**?

Reply:
I think it makes sense to move some of the description, such as adding to the end of what is there:  

When the `f2k` function is called, it creates a local variable `k`. The function does not return any values and does not alter `k` outside of its local copy. Therefore the original value of `k` remains unchanged.

I think it could also be useful to create a glossary entry for local variable and link to it there. The text could also differ somewhat if you think it could be explained better in a different way.

Reply:
Agreed. Does this mean we are moving towards removing **The Old Switcheroo**? I'll try and implement the changes you propose.

Reply:
Unless @maxim-belkin or  others in the community speaks up about why we should keep the exercise, it would be great for you to carry out the proposed changes. Thank you, @svaberg!

Reply:
I've carried out all the proposed changes but I have not yet removed **the Old Switcheroo**. I'll do that now in a separate commit.